integration_tests/src/test/resources/decimals.json

```diff
@@ -0,0 +1,12 @@
+{ "number": 0 }
+{ "number": 12 }
+{ "number": 12.0 }
+{ "number": 12. }
+{ "number": .34 }
+{ "number": +.34 }
+{ "number": -.34 }
+{ "number": 0.34 }
+{ "number": 12.34 }
+{ "number": 12.3456 }
+{ "number": 12.345678 }
```

Comment:
Same here it would be nice to have some numbers that would possibly round down instead of up if Spark was rounding.